admin/app/helpers/solidus_admin/permission_sets_helper.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+module SolidusAdmin
+  module PermissionSetsHelper
+    # @note This is exceptionally non-DRY, but so far it has proven to be the
+    #   safest way to reliably match on these PermissionSets. Other
+    #   optimizations that were tested resulted in too many false matches on
+    #   other non-stock-Solidus but similarly named PermissionSets.
+    # @param permission_sets [Array<Spree::PermissionSet>] an array of
+    #   PermissionSet objects to be organized into categories based on their
+    #   names.
+    # @param view_string [String] A string of your choice associated with "View"
+    #   or "Display" level permissions. Used when rendering the checkbox.
+    # @param edit_string [String] A string of your choice associated with "Edit"
+    #   or "Management" level permissions. Used when rendering the checkbox.
+    def organize_permissions(permission_sets:, view_string:, edit_string:)
+      return {} if permission_sets.blank?
+
+      permission_sets.each_with_object({}) do |permission, grouped_permissions|
+        case permission.name
+        when /OrderDisplay/i
```

Comment:
My only idea: we can try to document and encourage developers to create permission sets that end in "display" and "management" for view and edit permissions, and then make this helper more DRY by looking only for either `Display` or `Management` in the class name.